tests/unittests/test_boolean_conversion.py

```diff
@@ -0,0 +1,88 @@
+import unittest
+from tap_google_sheets.singer_transform import transform
+
+class TestBooleanDataType(unittest.TestCase):
+    def test_string_returned_for_non_boolean_columns(self):
```

Comment:
Add function comment for all the unit tests and also add one unit test for the `False` value scenario.

Reply:
Added comments